modules/installation-mirror-repository.adoc

```diff
@@ -31,107 +35,66 @@ x509: certificate relies on legacy Common Name field, use SANs or temporarily en
+PRODUCT_REPO='openshift'
+LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
 ifndef::openshift-origin[]
-[source,terminal]
-----
-$ PRODUCT_REPO='openshift-release-dev'
-----
-+
-For a production release, you must specify `openshift-release-dev`.
+RELEASE_NAME='ocp-release' <6>
 endif::[]
 ifdef::openshift-origin[]
-[source,terminal]
-----
-$ PRODUCT_REPO='openshift'
-----
+RELEASE_NAME='okd' <6>
 endif::[]
-
-.. Export the path to your registry pull secret:
-+
-[source,terminal]
-----
-$ LOCAL_SECRET_JSON='<path_to_pull_secret>'
-----
-+
-For `<path_to_pull_secret>`, specify the absolute path to and file name of the pull secret for your mirror registry that you created.
-
-.. Export the release mirror:
-+
+REMOVABLE_MEDIA_PATH='<path>' <7>
 ifndef::openshift-origin[]
-[source,terminal]
-----
-$ RELEASE_NAME="ocp-release"
-----
-+
-For a production release, you must specify `ocp-release`.
-endif::[]
-ifdef::openshift-origin[]
-[source,terminal]
-----
-$ RELEASE_NAME="okd"
-----
+ARCHITECTURE='<server_architecture>' <8>
 endif::[]
-
+set +a #shut off export of defined variables
+# alternatively set +o allexport
+#
+----
+<1> For `<release_version>`, specify the tag that corresponds to the version of {product-title} to install, such as `4.5.4`.
+<2> For `<local_registry_host_name>`, specify the registry domain name for your mirror repository, and for `<local_registry_host_port>`, specify the port that it serves content on.
+<3> For `<local_repository_name>`, specify the name of the repository to create in your mirror registry, such as `ocp4/openshift4`.
+<4> For a production release, you must specify `openshift-release-dev`.
+<5> For `<path_to_pull_secret>`, specify the absolute path to and file name of the pull secret for the mirror registry that you created.
 ifndef::openshift-origin[]
-.. Export the type of architecture for your server, such as `x86_64`:
-+
-[source,terminal]
-----
-$ ARCHITECTURE=<server_architecture>
-----
+<6> For a production release, you must specify `ocp-release`.
+endif::[]
+ifdef::openshift-origin[]
+<6> You must specify `okd`.
 endif::[]
+<7> For `<path>`, specify the full path, including the initial forward slash (/) character.
+<8> For `<server_architecture>`, specify the type of architecture for your server, such as `x86_64`.
 
-.. Export the path to the directory to host the mirrored images:
+.. To apply the environment variables, run the script:
 +
 [source,terminal]
 ----
-$ REMOVABLE_MEDIA_PATH=<path> <1>
+$ . ./environment_variables.sh
 ----
-<1> Specify the full path, including the initial forward slash (/) character.
 
-. Mirror the version images to the internal container registry:
-** If your mirror host does not have internet access, take the following actions:
-... Connect the removable media to a system that is connected to the internet.
+. Mirror the version images to the mirror registry:
+** If your workstation does not also host mirror registry, take the following actions:
```

Comment:
Actually per my understanding, in the whole mirror procedure, there should be 2 types of hosts. 

1. workstation. This host should have internet connectivity, user can download installation media on this host, also run oc command to do mirror operations. If this workstation can connect mirror registry in restrict network, then mirror images to internal mirror registry directly; If this workstation can not connect mirror registry either, then have to mirror to local removable media, then bring the removable media into restrict network, and upload images to mirror registry using some similar workstation in restrict network.
2. mirror registry. This host should be hosted in restrict network as a central registry for the following installation, only used as target registry for image mirror operations.

So it is better not mix "workstation" and "mirror registry" concept together in our doc, so that user can clearly understand the topology in this mirror procedure.